### ISSUE TITLE
different E-Mail Provider using resend instead of Sendgrid

### ISSUE THREAD

**Issue by Hybiekay** [digitalocean/supabase-on-do#46]
I've been facing authorization issues while trying to use Sendgrid after signing up. I have set up my Resend account and verifying my domain, and
Considering this, I'm interested in switching to a different email provider. Could anyone provide guidance on how to make this transition? Thank you!"

**Comment by pooriaarab** [comment]
Yes. Want to use https://github.com/postalserver/postal instead of Sendgrid. How can we do that?

@timothymamo

**Comment by JalalMitali** [comment]
yes we don't want SendGrid :)

**Comment by JalalMitali** [comment]
@Hybiekay could you find a solution. by the way to send grid signup issue I got the same!

**Comment by ndmikkelsen** [comment]
It is important to make this modular, as to be able to provide your own email provider via the terraform.tfvars file.
